src/Assets/ScriptWithBuiltDependenciesAsset.php

```diff
@@ -0,0 +1,68 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\Assets;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\Value\BuiltScriptDependencyArray as DependencyArray;
+use Throwable;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class ScriptWithBuiltDependenciesAsset
```

Comment:
💅  Same as https://github.com/woocommerce/google-listings-and-ads/pull/1488/files#r884728853 comment.